src/Core/ConstraintValidator/CleanHtmlValidator.php

```diff
@@ -57,7 +67,8 @@ public function validate($value, Constraint $constraint)
         $containsScriptTags = preg_match('/<[\s]*script/ims', $value) || preg_match('/.*script\:/ims', $value);
         $containsJavascriptEvents = preg_match('/(' . $this->getJavascriptEvents() . ')[\s]*=/ims', $value);
 
-        if ($containsScriptTags || $containsJavascriptEvents) {
+        $iframe = !$this->allowIframe && preg_match('/<[\s]*(i?frame|form|input|embed|object)/ims', $value);
```

Comment:
Please move the pattern to private const.

Reply:
Done